diff -up bullet-2.77/Extras/glui/CMakeLists.txt.extras-version bullet-2.77/Extras/glui/CMakeLists.txt
--- bullet-2.77/Extras/glui/CMakeLists.txt.extras-version	2010-03-06 16:23:36.000000000 +0100
+++ bullet-2.77/Extras/glui/CMakeLists.txt	2010-07-26 12:09:01.000000000 +0200
@@ -61,6 +61,8 @@ arcball.cpp			glui_button.cpp			glui_fil
 	
 )
 
+SET_TARGET_PROPERTIES(GLUI PROPERTIES VERSION ${BULLET_VERSION})
+SET_TARGET_PROPERTIES(GLUI PROPERTIES SOVERSION ${BULLET_VERSION})
 IF (BUILD_SHARED_LIBS)
   TARGET_LINK_LIBRARIES(GLUI ${GLUT_glut_LIBRARY} ${OPENGL_gl_LIBRARY} ${OPENGL_glu_LIBRARY})
 ENDIF (BUILD_SHARED_LIBS)
